guide/message_transformation/message_transformation.bal

```diff
@@ -0,0 +1,370 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/mime;
+import ballerina/io;
+import ballerina/mysql;
+//import ballerinax/docker;
+//import ballerinax/kubernetes;
+
+//connect the student details table
+endpoint mysql:Client testDB {
+    host: "localhost",
+    port: 3306,
+    name: "testdb",
+    username: "root",
+    password: "",
+    poolOptions: { maximumPoolSize: 5 },
+    dbOptions: { useSSL: false }
+};
+//connect the student's results details table
+endpoint mysql:Client testDB1 {
+    host: "localhost",
+    port: 3306,
+    name: "testdb1",
+    username: "root",
+    password: "",
+    poolOptions: { maximumPoolSize: 5 },
+    dbOptions: { useSSL: false }
+};
+
+//@kubernetes:Ingress {
+//    hostname:"ballerina.guides.io",
+//    name:"message_transformation",
+//    path:"/"
+//}
+//@kubernetes:Service {
+//    serviceType:"NodePort",
+//    name:"contentfilter"
+//}
+//@kubernetes:Service {
+//    serviceType:"NodePort",
+//    name:"validator"
+//}
+//@kubernetes:Service {
+//    serviceType:"NodePort",
+//    name:"enricher"
+//}
+//@kubernetes:Service {
+//    serviceType:"NodePort",
+//    name:"backend"
+//}
+//@kubernetes:Deployment {
+//    image:"ballerina.guides.io/message_transformation_service:v1.0",
+//    name:"ballerina-guides-message-transformation-service",
+//    baseImage:"ballerina/ballerina-platform:0.980.0",
+//    copyFiles:[{target:"/ballerina/runtime/bre/lib",
+//        source:"/home/saneth/Documents/ballerina/mysql-connector-java-5.1.46.jar"}]
+//}
+
+//@docker:Config {
+//    registry:"ballerina.guides.io",
+//    name:"message_transformation",
+//    tag:"v1.0",
+//    baseImage:"ballerina/ballerina-platform:0.980.0"
+//}
+//@docker:CopyFiles {
+//    files:[{source:"/home/saneth/Documents/ballerina/mysql-connector-java-5.1.46.jar",
+//        target:"/ballerina/runtime/bre/lib"}]
+//}
+//
+//@docker:Expose {}
+endpoint http:Listener contentfilterEP {
+    port:9090
+};
+endpoint http:Listener claimvaditadeEP {
+    port:9094
+};
+endpoint http:Listener contentenricherEP {
+    port:9092
+};
+endpoint http:Listener backendEP {
+    port:9093
+};
+//define endpoints for services
+endpoint http:Client validatorEP {
+    url: "http://localhost:9094/validator"
+};
+endpoint http:Client enricherEP {
+    url: "http://localhost:9092/enricher"
+};
+endpoint http:Client clientEP {
+    url: "http://localhost:9093/backend"
+};
+
+
+
+//define the global variables
+public json payload1;
+public json payload2;
+
+//service for the content filter pattern
+service<http:Service> contentfilter bind contentfilterEP {
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/"
+    }
+    contentfilter(endpoint caller, http:Request req) {
+        http:Request filteredreq = req;
```

Comment:
Shall we rename the variable name with camelcase (`filteredReq`)? Check all the other variables as well. (`validatededreq`, `enrichedreq`, ...)

Reply:
Done